compiler/p/codegen/OMRRegisterDependency.cpp

```diff
@@ -179,15 +180,16 @@ OMR::Power::RegisterDependencyConditions::RegisterDependencyConditions(
       TR::RealRegister::RegNum regNum = (TR::RealRegister::RegNum)cg->getGlobalRegister(child->getGlobalRegisterNumber());
 
       TR::RealRegister::RegNum highRegNum;
+      TR_GlobalRegisterNumber validHighReg = TR::TreeEvaluator::getHighGlobalRegisterNumberIfAny(child, cg); 
```

Comment:
I find the use of the suffix `Reg` on the variable name confusing in this context.  In other contexts in this function and file it is used to hold `TR::Register`s.  Perhaps `validHighRegNum` would be a better name.